config/samples/cluster_v1alpha2_cluster.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cluster.k8s.io/v1alpha2
```

Comment:
Same thing with the v1alpha2 samples, they are invalid and can likely just be removed.